balancer/frontend-v2
# Move isDeep and isBoosted from usePools to config files

# Description

- Moving the list of deep and boosted pools from the usePools file to individual config files. 

## Type of change


## How should this be tested?

- Ensure that pools that are deep show the correct breakdown of assets on their individual pool page. 
- Ensure pools in the boosted list show their APR correctly with `Boosted APR` in the list. 

## Checklist:
